patches/new-compilers.patch

```diff
@@ -0,0 +1,612 @@
+diff --git a/gfx/angle/checkout/include/GLSLANG/ShaderVars.h b/gfx/angle/checkout/include/GLSLANG/ShaderVars.h
+index 4b76d33..6adb28d 100644
```

Comment:
Is this a patch that we are adding? Do you mind describing the purpose? I think "fix-build-on-newer-compilers.patch" was probably a better name.

Reply:
I thought this patch was not needed, but I needed to add part of it again so I did it lazily.